cloudformation/build-bundle-test-workflow.yml

```diff
@@ -0,0 +1,59 @@
+Description: Template for setting up resources for the build-bundle-test workflows
+Resources:
+  BuildBucket:
+    Type: "AWS::S3::Bucket"
+  BuildUser:
+    Type: "AWS::IAM::User"
+    Properties:
+      UserName: builder
+      Policies:
+        - PolicyName: S3_Write_Access
```

Comment:
Rename to `S3_Build_Write_Access` for consistency.

Reply:
Done